pkg/cli/common.go

```diff
@@ -95,6 +95,10 @@ var (
 			Value: "",
 			Usage: "use `[username[:password]]` for accessing the registry",
 		},
+		cli.BoolFlag{
+			Name:  "disable-content-trust",
+			Usage: "This is a Docker specific option and is a NOOP for Buildah",
```

Comment:
Don't say Buildah, since this will also be used in podman.
Just remove for Buildah.

Reply:
@rhatdan ack, good point.